packages/vscode-dependencies-validation/src/extension.ts

```diff
@@ -1,27 +1,96 @@
-import { Uri, ExtensionContext, workspace, window } from "vscode";
 import {
-  NPMDependencyIssue,
+  Uri,
+  ExtensionContext,
+  workspace,
+  window,
+  languages,
+  DiagnosticCollection,
+  OutputChannel,
+  commands,
+  CodeActionKind,
+} from "vscode";
+import {
+  NpmLsResult,
   findDependencyIssues,
 } from "@sap-devx/npm-dependencies-validation";
+import { NPMIssuesActionProvider } from "./npmIssuesActionProvider";
+import { fixAllDepIssuesCommand } from "./commands";
+import { FIX_ALL_ISSUES_COMMAND } from "./constants";
+import { refreshDiagnostics } from "./diagnostics";
+
+const PACKAGE_JSON = "package.json";
+const PACKAGE_JSON_PATTERN = `**​/${PACKAGE_JSON}`;
+const extName = "vscode-dependency-validation";
```

Comment:
is not this name (or similar) available in https://code.visualstudio.com/api/references/vscode-api#ExtensionContext
provided to the `activate` method (I think under context.extension.id)

Reply:
Done